R/db-odbc-mssql.R

```diff
@@ -100,3 +94,14 @@
   )
   DBI::dbExecute(con, sql)
 }
+
+mssql_is_null <- function(x, sql_context){
+  if ( !is.null(sql_context$mutate) | !is.null(sql_context$arrange) ) {
```

Comment:
Can you please also add a comment documenting the issue? (My comments in slack about bit vs boolean will probably be most helpful to me)